crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po

```diff
@@ -145,20 +145,30 @@ msgstr ""
 msgid "Primary classification"
 msgstr "Clasificación principal"
 
-#: forms.py:895
+#: forms.py:897
 msgid "Assigned to"
 msgstr "Asignado a"
 
+#: forms.py:1000
+#, fuzzy
+#| msgid "Date can not be in the future."
+msgid "Create date cannot be in the future."
+msgstr "La fecha no puede ser en el futuro."
+
+#: forms.py:1141
+msgid "Comment cannot be empty"
+msgstr ""
```

Comment:
This one is on me. But I think it's OK since we aren't doing spanish translations for the back end.